components/UrlPicker/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * WordPress dependencies
+ */
+ import { __ } from '@wordpress/i18n';
+ import { useState } from '@wordpress/element';
+ import {
+   KeyboardShortcuts,
+   Popover,
+   Button,
+   Tooltip,
+ } from '@wordpress/components';
+ import {
+   PlainText,
+   __experimentalLinkControl as LinkControl,
+ } from '@wordpress/block-editor';
+ import { rawShortcut } from '@wordpress/keycodes';
+ import { debounce } from 'lodash';
+ import { searchablePostTypes } from './helper';
```

Comment:
@helen Seems this function is missing here. Would you be able to also add it to this file? I cam take care of the refactor then :)

Reply:
Added it as an inline const :)